README.md

```diff
@@ -4,6 +4,9 @@
 # BlueChi: A systemd service controller for multi-node environments
 
 [![Copr build status](https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/bluechi-snapshot/package/bluechi/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/bluechi-snapshot/package/bluechi/)
+
```

Comment:
Please remove this empty line, it would be better to have all badges on the same line

Reply:
I wanted to group the test-related badges and separate the copr build one. I can remove the line, though.

Reply:
Changed.

Reply:
:+1:  Thanks